datasets/newsqa/README.md

```diff
@@ -0,0 +1,189 @@
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+Configuration: combined-csv
+ - `story_id`: An identifier of the story
+- `story_text`: text of the story
+- `question`: A question about the story.
+- `answer_char_ranges`: The raw data collected for character based indices to answers in story_text. E.g. 196:228|196:202,217:228|None. Answers from different crowdsourcers are separated by |, within those, multiple selections from the same crowdsourcer are separated by ,. None means the crowdsourcer thought there was no answer to the question in the story. The start is inclusive and the end is exclusive. The end may point to whitespace after a token.
+
+Configuration: combined-csv
+- `storyId`: An identifier of the story.
+- `text`: Text of the story
+- `type`: Split type - train, validation or test
+- `questions`: A list containing the following. 
+  - `q`: A question
+  - `isAnswerAbsent`: Proportion of crowdsourcers that said there was no answer to the question in the story.
+  - `isQuestionBad`: Proportion of crowdsourcers that said the question does not make sense.
+  - `consensus`: The consensus answer. Use this field to pick the best continuous answer span from the text. If you want to know about a question having multiple answers in the text then you can use the more detailed "answers" and "validatedAnswers". The object can have start and end positions like in the example above or can be {"badQuestion": true} or {"noAnswer": true}. Note that there is only one consensus answer since it's based on the majority agreement of the crowdsourcers.
+  - `s`: start of the answer
+  - `e`: end of the answer
+  - `badQuestion`: The validator said that the question did not make sense.
+  - `noAnswer`: The crowdsourcer said that there was no answer to the question in the text.
+  - `answers`: The answers from various crowdsourcers.
+    - `sourcerAnswers`: The answer provided from one crowdsourcer.
+      - `s`: start
+      - `e`: end
+      - `noAnswer`: The crowdsourcer said that there was no answer to the question in the text.
+  - `validated_answers`: The answers from the validators.
+    - `sourcerAnswers`: The answer provided from one crowdsourcer.
+      - `s`: start
+      - `e`: end
+      - `noAnswer`: The crowdsourcer said that there was no answer to the question in the text.
+      - `count`: The number of validators that agreed with this answer.
+
+Configuration: split
+ - `story_id`: An identifier of the story
+- `story_text`: text of the story
+- `question`: A question about the story.
+- `answer_token_ranges`: Word based indices to answers in story_text. E.g. 196:202,217:228. Multiple selections from the same answer are separated by ,. The start is inclusive and the end is exclusive. The end may point to whitespace after a token.
```

Comment:
all this must go inside the `Data Fields` section instead of `Data Instances`

In `Data Instances` we expect an example from the dataset. You can get one by using `load_dataset` and take the first sample with `dataset["train"][0]` for example